README.md

```diff
@@ -37,3 +37,15 @@ Code is formatted by [Black](https://github.com/ambv/black).
 ## License
 
 [MIT](https://github.com/kiwicom/gitlab-unfurly/blob/master/LICENSE)
+
+## FAQ
+
+1. **How to make unfurly ignore my urls (= not create previews)?**  
+Append `?no_unfurl` to the url.
+
+2. **Is there a way to disable unfurly for the whole post?**  
+Unfortunately no. Unfurly receives links from slack 1-by-1. Therefore it has no knowledge about
+links being part of the same post.
+
+3. **But this is very tiresome.**  
+We know. PRs welcome.
```

Comment:
Well, thanks for taking the time to submit this, but could you make this a bit less passive aggressive? It also doesn't quite follow since I cannot imagine how a PR could introduce a less tiresome process.

Reply:
Was trying to keep the spirit of communication from relevant issue. Will adjust it then.

Reply:
Adjusted.